src/geometry/point_conversion.rs

```diff
@@ -67,3 +76,57 @@ where
         Self::from_coordinates(::convert_unchecked(coords))
     }
 }
+
+
+
+#[cfg(feature = "mint")]
+macro_rules! impl_from_into_mint_1D(
+    ($($NRows: ident => $PT:ident, $VT:ident [$SZ: expr]);* $(;)*) => {$(
+        impl<N> From<mint::$PT<N>> for Point<N, $NRows>
+        where N: Scalar,
+              DefaultAllocator: Allocator<N, $NRows> {
```

Comment:
This `DefaultAllocator` bound can be removed. This is never necessary when the dimension of the vector/matrix is given as a concrete type.